fio.1

```diff
@@ -1857,6 +1857,12 @@ than normal.
 When hipri is set this determines the probability of a pvsync2 I/O being high
 priority. The default is 100%.
 .TP
+.BI (pvsync2,libaio,io_uring)nowait
+Set RWF_NOWAIT on I/O, indicating to the kernel to do not wait if request
+cannot be executed immediately. (supported from the 4.14 Linux kernel)
+
+It is useful to also use ignore_error=EAGAIN when using this option.
```

Comment:
This should just match the same text as the HOWTO (but with different formatting if necessary)